app/classes/Transvision/AnalyseStrings.php

```diff
@@ -44,7 +44,11 @@ public static function differences($tmx_source, $tmx_target, $repo, $ignored_str
 
         if (Strings::startsWith($repo, 'gaia')) {
             $patterns = [
-                'l10njs'     => '/\{\{\s*([a-z0-9_]+)\s*\}\}/iu', // {{foobar2}}
+                'l10njs' => '/\{\{\s*([a-z0-9_]+)\s*\}\}/iu', // {{foobar2}}
+            ];
+        } elseif ($repo == 'firefox_ios') {
+            $patterns = [
+                'ios' => '/(%(?:[0-9]+\$){0,1}@)/i', // %@, but also %1$@, %2$@, etc.
             ];
         } else {
             $patterns = [
```

Comment:
nit: I am not a fan of multiple if/elseif/else statements, I tend to think that a switch is more readable:
```php
switch($repo) {
    case Strings::startsWith($repo, 'gaia'):
        $patterns = [
            'l10njs' => '/\{\{\s*([a-z0-9_]+)\s*\}\}/iu', // {{foobar2}}
        ];
        break;
    case 'firefox_ios':
        $patterns = [
            'ios' => '/(%(?:[0-9]+\$){0,1}@)/i', // %@, but also %1$@, %2$@, etc.
        ];
        break;
    default:
        $patterns = [
            'dtd'        => '/&([a-z0-9\.]+);/i',                        // &foobar;
            'printf'     => '/(%(?:[0-9]+\$){0,1}(?:[0-9].){0,1}(S))/i', // %1$S or %S. %1$0.S and %0.S are valid too
            'properties' => '/(?<!%[0-9])\$[a-z0-9\.]+\b/i',             // $BrandShortName, but not "My%1$SFeeds-%2$S.opml"
            'l10njs'     => '/\{\{\s*([a-z0-9_]+)\s*\}\}/iu',            // {{foobar2}} Used in Loop and PDFViewer
        ];
        break;
}
```